catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/OrderRequestImplTest.java

```diff
@@ -678,7 +757,20 @@ private Destination getBadHttpDestination() {
         return destination;
     }
 
+    private Destination getGoodEmailDestination() {
+        mockEmailSender = new EmailSender("donotreply@test.com", "connexta-com.mail.protection.outlook.com");
+        mockEmailSender = mock(EmailSender.class);
```

Comment:
More of a lack of intent in the face of trying various interesting combinations. Will fix :)